setup.py

```diff
@@ -27,7 +27,7 @@
     assert False, "Error: Could not open '%s' due %s\n" % (filepath, error)
 
 INSTALL_REQUIRE = [
-    "torch>=1.11",
+    "torch>=1.11,<2.3",
```

Comment:
I think it make sense to pin the torch version for IPEX tests only, the IPEX version should always match the torch version (it's not related to optimum-intel)

Reply:
thanks makes sense